delta/plugins/elasticsearch/src/main/resources/defaults/default-mapping.json

```diff
@@ -1,170 +1,141 @@
 {
   "properties": {
     "@type": {
-      "type": "keyword",
```

Comment:
This is introducing a breaking change, we need to introduce a multi-field where the full text search would be run in the subfield.
It may make the full text search easier as we can only check on those fields (*.fulltext maybe) ?

Reply:
The nice thing about this is that we no longer need to match on `_*` but `*.fulltext` and so we don't match on ES metadata